{"name": "Отказ перевода: недостаточно средств", "status": "passed", "steps": [{"name": "Создаем счет для Eric и Clyde", "status": "passed", "start": 1758556839373, "stop": 1758556839374}, {"name": "Добавляем этих господ в сессию", "status": "passed", "start": 1758556839374, "stop": 1758556839374}, {"name": "Сохраняем изменения", "status": "passed", "start": 1758556839374, "stop": 1758556839931}, {"name": "Пытаемся совершить перевод 2000 ед. - ожидаем ошибку \n        ValueError", "status": "passed", "steps": [{"name": "Перевод 2000 ед. со счета 'Eric1e0e27' на счет 'Clydea8d405'", "status": "broken", "statusDetails": {"message": "ValueError: Недостаточно средств на счете\n", "trace": "  File \"/Users/maxnovo/Desktop/Kokos AQA/Cinescope/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/maxnovo/Desktop/Kokos AQA/Cinescope/tests/api/test_transactions.py\", line 26, in transfer_money\n    raise ValueError(\"Недостаточно средств на счете\")\n"}, "steps": [{"name": " Получаем счета", "status": "passed", "start": 1758556840484, "stop": 1758556840759}, {"name": "Проверяем, что на счете достаточно средств", "status": "broken", "statusDetails": {"message": "ValueError: Недостаточно средств на счете\n", "trace": "  File \"/Users/maxnovo/Desktop/Kokos AQA/Cinescope/tests/api/test_transactions.py\", line 26, in transfer_money\n    raise ValueError(\"Недостаточно средств на счете\")\n"}, "start": 1758556840759, "stop": 1758556840759}], "parameters": [{"name": "session", "value": "<sqlalchemy.orm.session.Session object at 0x10638c5c0>"}, {"name": "from_account", "value": "'Eric1e0e27'"}, {"name": "to_account", "value": "'Clydea8d405'"}, {"name": "amount", "value": "2000"}], "start": 1758556840484, "stop": 1758556840762}], "start": 1758556839931, "stop": 1758556840762}, {"name": "Проверяем повторно баланс из БД после отката операции", "status": "passed", "start": 1758556840762, "stop": 1758556840762}, {"name": "Чистим данные после себя", "status": "passed", "start": 1758556840763, "stop": 1758556840763}, {"name": "Сохраняем изменения", "status": "passed", "start": 1758556840763, "stop": 1758556841174}], "attachments": [{"name": "log", "source": "579b8a44-bad9-41c4-946d-fbc1e556a0da-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "917be9cd-a895-4a47-9ea1-558a47330eea-attachment.txt", "type": "text/plain"}], "start": 1758556839373, "stop": 1758556841175, "uuid": "c3a57cf0-27f4-4075-85f1-0ce791756736", "historyId": "9a5455919489f8b309d1755e154790ec", "testCaseId": "9a5455919489f8b309d1755e154790ec", "fullName": "tests.api.test_transactions#test_accounts_transfer_insufficient_funds", "labels": [{"name": "feature", "value": "Тестирование транзакций между счетами"}, {"name": "story", "value": "Отказ в переводе при недостатке средств"}, {"name": "severity", "value": "critical"}, {"name": "epic", "value": "Тестирование транзакций"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_transactions"}, {"name": "host", "value": "MacBook-Pro.local"}, {"name": "thread", "value": "48839-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_transactions"}], "titlePath": ["tests", "api", "test_transactions.py"]}